src/Sylius/Bundle/AddressingBundle/Form/Type/AddressType.php

```diff
@@ -78,6 +81,7 @@ public function configureOptions(OptionsResolver $resolver): void
                     return $this->validationGroups;
                 },
                 'shippable' => false,
+                'add_build_address_form_subscriber' => true,
```

Comment:
:thinking: Add information about this option to the readme, WDYT?

Reply:
1) It's minor change
2) It doesn't change a lot, as we added it for the new feature, previously behavior has been kept
3) We might pivot into separate form types in the meantime